src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/UploadDecisionLetterHandler.java

```diff
@@ -69,20 +70,28 @@ public boolean canHandle(
                 .map(Optional::get)
                 .collect(Collectors.toList());
 
```

Comment:
I case on empty list you can return from handler because no further work is needed.
That will allow you to keep integration tests without proposed changes

Reply:
This makes sense, thanks for this. I now check for empty list and return from handler, which allows me to revert the changes to integration tests.

I have also removed the following, which I don't think is necessary, if you could please confirm. 

`.orElseThrow(() -> new IllegalStateException("upload notice decision is not present"))`